CMakeLists.txt

```diff
@@ -123,11 +123,13 @@ option(FLB_IN_FORWARD         "Enable Forward input plugin"         Yes)
 option(FLB_IN_HEALTH          "Enable Health input plugin"          Yes)
 option(FLB_IN_HTTP            "Enable HTTP input plugin"            Yes)
 option(FLB_IN_MEM             "Enable Memory input plugin"          Yes)
+option(FLB_IN_KAFKA           "Enable Kafka input plugin"           No)
```

Comment:
Maybe should be on now or before merge?

Reply:
Not sure. I followed the existing FLB_OUT_KAFKA and left disabled by default. Should I enable both by default?